static/src/javascripts/projects/common/modules/ui/autoupdate.js

```diff
@@ -163,13 +167,28 @@ const autoUpdate = (opts) => {
                     }
                 }
 
-                setUpdateDelay();
+                pollUpdates && setUpdateDelay();
```

Comment:
might not need this

Reply:
@abeddow91

Reply:
We do need this in order to block the timed updates if we're on a deadblog.

Reply:
This is now removed.